frontend/public/components/build-pipeline.tsx

```diff
@@ -40,9 +40,10 @@ const BuildSummaryStatusIcon: React.SFC<BuildSummaryStatusIconProps> = ({ status
 
 const BuildLogLink: React.SFC<BuildLogLinkProps> = ({ obj }) => {
   const link = getJenkinsLogURL(obj);
-  return link && <div className="build-pipeline__link">
-    <a href={link} className="build-pipeline__log-link" target="_blank" rel="noopener noreferrer">View Log</a>
-  </div>;
+  return link ?
+    <div className="build-pipeline__link">
+      <a href={link} className="build-pipeline__log-link" target="_blank" rel="noopener noreferrer">View Log</a>
+    </div> : <LoadingInline/>;
```

Comment:
Not sure if we need the loading animation, but will wait for input from @openshift/team-ux-review

Reply:
Since the link won't be available until the build actually starts, I thought it made sense to have something there indicating that there is more to come.

Reply:
Right, understood. I'm just not sure if it's more distracting than helpful. Animations tend to draw your eye, and it could be in that state for a while.